core/src/test/scala/unit/kafka/coordinator/ConsumerCoordinatorResponseTest.scala

```diff
@@ -232,6 +235,18 @@ class ConsumerCoordinatorResponseTest extends JUnitSuite {
   }
 
   @Test
+  def testCommitOffsetFromUnknownGroup() {
+    val groupId = "groupId"
+    val consumerId = "consumer"
+    val generationId = 1
```

Comment:
Can you add another test going into the other branch you added (unknown group and generation id < 0)?